platform/linux-generic/odp_classification.c

```diff
@@ -1782,3 +1782,95 @@ uint64_t odp_pmr_to_u64(odp_pmr_t hdl)
 {
 	return _odp_pri(hdl);
 }
+
+static
+void print_cos_ident(struct cos_s *cos)
+{
+	ODP_PRINT("%s(%d)\n", cos->name, (int)cos->index);
+}
+
+static
+void print_queue_ident(odp_queue_t q)
+{
+	odp_queue_info_t info;
+
+	if (odp_queue_info(q, &info))
+		ODP_PRINT("%" PRIx64, odp_queue_to_u64(q));
+	else
+		ODP_PRINT("%s", info.name);
```

Comment:
Name is optional for both CoS and queue. In both cases, handle pointer could be printed when the name is missing.

Reply:
For cos, handle to u64 is just index+1, so it looks a little goofy:
```
1(0)
    queue: 7f7df0d98180
    rules: PMR_SIP_ADDR: 0a0a0a00 ffffff00 -> 2(1)
2(1)
    queue: 7f7df0d98300
```

Reply:
_odp_cos_entry_from_idx() returns CoS pointer, if pointer is preferred over handle.

However, maybe CoS index and (optional) name are enough to identify CoS line in the table. CoS line would be either e.g. "cos: (index)" or "cos: name (index)". I think it's easy to follow "(index)" tags in the table. CoS names don't necessarily mention "cos".

```
Classifier info
---------------
cos: DefaultCos(0)
    queue: DefaultQueue
    rules: PMR_SIP_ADDR: 0a0a0a00 ffffff00 -> (1)
           PMR_SIP_ADDR: c0a80000 ffff0000 -> (2)
           PMR_CUSTOM_FRAME: offset:48 0123456789abcdef ffffffffffffffff -> mark:7 CoSqueue2(3)
cos: (1)
    queue: queue1
cos: (2)
    queue: queue2
cos: CoSqueue2(3)
    queue: queue2
   
```

Reply:
Done. I also changed the number in paranthesis from internal cos_s.index to odp_cos_to_u64(). Please check.